lib/config/migrations/custom/schedule-migration.ts

```diff
@@ -85,6 +86,7 @@ export class ScheduleMigration extends AbstractMigration {
         if (schedules[i].endsWith('days')) {
           schedules[i] = schedules[i].replace('days', 'day');
         }
+        schedules[i] = massageCronSchedule(schedules[i]);
```

Comment:
@rarkins decided to not massage four part cron expressions and never expected to have them working. do we have those inside our docs?

Reply:
I will add if they aren't. We maybe unintentionally allowed them because `cron-parser` did.